<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>class_new</class>
 <widget class="QDialog" name="class_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Class</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>53</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="alignmentBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Alignment:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="gold1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting Gold: </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="gold2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="gold3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hit Die:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="hitdie1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>First Level Points:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="firstlvlpts1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="firstlvlpts2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Level Up Points:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="lvlpts1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="descriptBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>246</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>205</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>INT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>INT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
